Source/WebKit/Shared/libwpe/WebEventFactory.cpp

```diff
@@ -31,6 +31,10 @@
 #include <cmath>
 #include <wpe/wpe.h>
 
+#if ENABLE(TOUCH_EVENTS)
+#include <wtf/HashMap.h>
+#endif
```

Comment:
Please don't guard includes unless you have to. It just leads to future build failures:

```suggestion
#include <wtf/HashMap.h>
```

Reply:
Okay.